src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConsumerConfig.java

```diff
@@ -93,6 +99,9 @@
                 new ConcurrentKafkaListenerContainerFactory<>();
         factory.setConsumerFactory(newMainConsumerFactory());
         factory.getContainerProperties().setAckMode(ContainerProperties.AckMode.MANUAL_IMMEDIATE);
+        factory.setErrorHandler((exception, data) -> {
+            slackMessagingService.sendDeserializationErrorMassage(data.topic(), data.partition(),  data.offset());
```

Comment:
we'd want to log out the exception message to our own logs as well as sending to slack

Reply:
I was going to but it will mean putting a logger into the config class.

Reply:
Found another way round it.